client/src/components/header/TodoLogin.vue

```diff
@@ -1,16 +1,35 @@
 <template>
-  <div class="mb-8 mt-20 flex items-center justify-between">
+  <div class="mb-8 mt-20 flex w-72 items-center justify-between sm:w-[40rem]">
```

Comment:
why don't you use the  w-40 instead of the custom one?

Reply:
Here I cannot use built-in class, because tailwind doesn't have built-in class for it sadly. w-40 sets width to 10 rem.

Reply:
what if you define it in tailwind config? 🤔